_posts/2015-01-19-6to5-sublime.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "6to5 Sublime support"
+author: Andres Suarez
+date:   2015-01-19 10:40:00
+categories: announcements
+share_text: "6to5 Sublime support"
+---
+
+We are pleased to announce <a href="https://github.com/6to5/6to5-sublime/" target="_blank">6to5-sublime</a>, a set of language syntax definitions for modern JavaScript with JSX extensions, as well as snippets for common React constructs (like lifecycle methods) in ES6. It's available as <a href="https://packagecontrol.io/packages/6to5" target="_blank">6to5</a> on <a href="https://packagecontrol.io" target="_blank">Package Control</a>. Eliminating barriers to ES6+ adoption is not just about having the most feature-ready transpiler – it's also about providing the right environment in which to write ES6+ JavaScript.
```

Comment:
Could you get rid of the `target="_blank"`'s and just use the markdown syntax for links?